terminology/terminology_index.adoc

```diff
@@ -0,0 +1,207 @@
+// vim: set syntax=asciidoc:
+[[Terminology]]
+== Terminology
+:data-uri:
+:homepage https://github.com/projectatomic/container-best-practices:
+
+=== Dictionary
+
+When discussing containerization, it’s important to have a solid grasp on the related vocabulary. One of the challenges people have is that many of the following terms
+are used interchangeably.It can be confusing, especially for newcomers.
+
+The goal of this section is to clarify these terms, so that we can speak the same language.
+
+==== Container Image
+
+Container image is a binary that includes all of the requirements for running a container, as well as metadata describing its needs and capabilities. You can think of it as a packaging technology. Containers only have access to resources defined in the image,
```

Comment:
could we please say "filesystem tree" instead of binary?

Edit: I would rewrite it into:

```
Container image is a filesystem tree that includes all of the requirements for running a container, as well as metadata describing the content. You can think of it as a packaging technology.
```

I did not follow what you meant with runtime sentence: I would not talk about it in context of images. The fact is that container images itself don't say anything about runtime (except for `run` label maybe).